deisctl/cmd/platform.go

```diff
@@ -11,12 +11,18 @@ import (
 
 // InstallPlatform loads all components' definitions from local unit files.
 // After InstallPlatform, all components will be available for StartPlatform.
-func InstallPlatform(b backend.Backend, checkKeys func() error) error {
+func InstallPlatform(b backend.Backend, checkKeys func() error, stateless bool) error {
 
 	if err := checkKeys(); err != nil {
 		return err
 	}
 
+	if stateless {
+		fmt.Println("Warning: With a stateless control plane, `deis logs` will be unavailable.")
+		fmt.Println("Additionally, components will need to be configured to use external persistent stores.")
+		fmt.Println("See the official Deis documentation for details on running a stateless control plane.")
```

Comment:
I confirmed that controller refuses to start without `deis-store-volume`:
```
Jul 01 22:27:33 deis-02 fleetd[1050]: ERROR manager.go:129: Failed to trigger systemd unit deis-controller.service start: Unit deis-store-volume.service failed to load: No such file or directory.
```

Going to see if making controller just `Wants` it resolves the issue.